packages/admin-ui/src/components/Filter/filter.tsx

```diff
@@ -0,0 +1,84 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import { Button } from '../Button'
+
+import { VisuallyHidden } from '../VisuallyHidden'
+import type {
+  FilterItem,
+  UseMultipleFilterReturn,
+} from './useMultipleFilterState'
+import { PopoverDisclosure } from 'reakit/Popover'
+import { PopoverFooter, Popover } from './popover'
+import { tag } from '@vtex/admin-ui-react'
+import { IconCaretUp } from '@vtex/phosphor-icons'
+import { MultipleSelectContent } from './multiple-select-content'
+
+export function Filter(props: FilterProps) {
+  const { state } = props
+  const { onClear, onApply, popover, selectedValues, label, labelProps } = state
+
+  const selectedItemsLabel =
+    selectedValues.length &&
+    `${selectedValues[0]}${
+      selectedValues.length > 1 ? `, +${selectedValues.length - 1}` : ''
+    }`
+
+  return (
+    <>
+      <VisuallyHidden>
+        <div {...labelProps}>{label}</div>
+      </VisuallyHidden>
+      <Button
+        as={PopoverDisclosure}
+        state={popover}
+        csx={{
+          bg: '$secondary',
+          color: '$secondary',
+          ':hover': {
+            // correct color \/
+            // bg: colors.gray10,
```

Comment:
Those commented line should stay or should be removed?

Reply:
I'm currently talking to davis to add the color as a token and will be doing that soon. Until then I'll leave the comment so I can remember the right color 😅 I'll remove it before merging

Reply:
It's done now